dtfs-central-api/src/helpers/payload.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Validates payload against template
+ * @param {Object} payload Payload to be validated
+ * @param {Object} template Template to be used for validation
+ * @returns {Boolean} Whether payload matches template
+ */
+const payloadVerification = (payload, template) => {
+  if (Object.keys(payload).length > 0 && Object.keys(template).length > 0) {
+    // 1. Properties key validation
+    const payloadKeys = Object.keys(payload);
+    const templateKeys = Object.keys(template);
+
+    const propertiesExists = templateKeys.every((propertyKey) => payloadKeys.includes(propertyKey));
```

Comment:
This checks that all the keys from the template exist, but it doesn't check that there aren't any unexpected keys in the payload so the user can still insert extra data into the db

Reply:
Have added.